src/Html/Builder.php

```diff
@@ -133,8 +186,9 @@ public function parameterize($attributes = [])
         $values       = [];
         $replacements = [];
 
+
         foreach (array_dot($parameters) as $key => $value) {
-            if ($this->isCallbackFunction($value, $key)) {
+            if ($this->isCallbackFunction($value, $key) || $this->isJavascriptFunction($value, $key)) {
```

Comment:
@OzanKurt I think this is just the line you need along with the `isJavascriptFunction` function for the new PR. Thanks!